<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>picsaveClass</class>
 <widget class="QMainWindow" name="picsaveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>picsave</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelServerAddr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>监控服务器地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServerAddr">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPicDir">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图片保存位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPicDir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSnapshotElapse">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>抓拍时间间隔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditElapse">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分钟</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelState">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最近抓拍时间 2015年5月20日 14:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenDir">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChooseNewDir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择新位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoRun">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开机自动运行</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="picsave.qrc"/>
 </resources>
 <connections/>
</ui>
